grails-forge-core/src/main/java/org/grails/forge/feature/other/MicronautHttpClient.java

```diff
@@ -61,7 +61,14 @@ public String getDocumentation() {
     public void apply(GeneratorContext generatorContext) {
         generatorContext.addDependency(Dependency.builder()
                 .groupId("io.micronaut")
-                .artifactId("micronaut-http-client")
+                .lookupArtifactId("micronaut-http-client")
+                .compile());
+
+
+        // micronaut-http-client no longer provides the jackson implementation
+        generatorContext.addDependency(Dependency.builder()
+                .groupId("io.micronaut.serde")
+                .lookupArtifactId("micronaut-serde-jackson")
                 .compile());
```

Comment:
Could this be `.runtime()`

Reply:
I am going to leave it for now, since I am not 100% sure runtime will cover all standard use cases and since this is an optional feature.